docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -368,10 +368,15 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-nullability-completeness ")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unneeded-internal-declaration ")
 ```
 
-### How to mute warnings for specific lines of code (NOT RECOMMENDED)
-In rare situations where the warnings cannot be fixed nor muted via regular attempts, one of the last things you can try is to decorate your code with the `#pragma clang diagnostic` macro. Beware that `#pragma`s are not part of the C++ standard and strongly depend on the compiler's implementation. That is to say, this solution is neither stable nor elegant.
+### Mute warnings from specific code blocks
+
+:::caution
+The approach presented here is *not* recommended, because it is *not* reliable and hence considered your last approach.
```

Comment:
considered as ...?

Reply:
Yeah, "considered as" is grammatical. But "as" can usually be omitted.

Reply:
However, I'll change the the sentence a bit: 

The approach presented here is *not* recommended and considered your last approach, because it is *not* reliable.

Thanks for pointing this out.